extras/indexing/src/main/java/org/apache/rya/indexing/accumulo/entity/EntityOptimizer.java

```diff
@@ -69,10 +72,8 @@ public EntityOptimizer(RdfCloudTripleStoreConfiguration conf) {
                 eval = new AccumuloSelectivityEvalDAO(conf, ConfigUtils.getConnector(conf));
                 ((AccumuloSelectivityEvalDAO)eval).setRdfEvalDAO(new ProspectorServiceEvalStatsDAO(ConfigUtils.getConnector(conf), conf));
                 eval.init();
-            } catch (AccumuloException e) {
-                e.printStackTrace();
-            } catch (AccumuloSecurityException e) {
-                e.printStackTrace();
+            } catch (final AccumuloException | AccumuloSecurityException | TableExistsException e) {
+                LOG.warn("A problem was encountered while constructing the EntityOptimizer.", e);
```

Comment:
I don't know what else you'd say here. The chained exception will have the details.